cassandra/util.py

```diff
@@ -12,7 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import with_statement
```

Comment:
Per [PEP-343](https://peps.python.org/pep-0343/) this became standard syntax in 2.6... this could've been removed long ago. :(